### PR TITLE
Atlassian Update.

### PR DESCRIPTION
**Pull request by Spectroxx**
**Atlassian Update**
**Updated**:
- Difficulty Easy >> Medium
- Notes
- Secondary URL

**Notes:**
If the account has an existing subscription active or cancelled you are required to designate another user as the primary billing and  and technical contact. Atlassian will **not** let you delete your account otherwise.

Source Image

### CODE REVIEW

**Comment by tupaschoal** [comment]
Hello @Spectroxx , was it not applicable? Just want to understand why you closed it

**Comment by Spectroxx** [comment]
@tupaschoal I must've done that on accident. My apologies lol

I just noticed an error, let me fix it real quick whilst I restore the branch.

File: _data/sites.json [hunk 1]
```diff
@@ -735,9 +735,11 @@
     {
         "name": "Atlassian",
         "url": "https://id.atlassian.com/manage/close-account",
-        "difficulty": "easy",
+        "difficulty": "medium",
+        "notes": "You needs to visit my.atlassian.com and designate a different user as a billing and technical contact if your account is the primary billing contact for one or more products",
```
Comment (tupaschoal) [inline _data/sites.json:739]:
```suggestion
        "notes": "You need to visit my.atlassian.com and designate a different user as a billing and technical contact if your account is the primary billing contact for one or more products",
```

**Comment by tupaschoal** [comment]
Thanks for improving Atlassian's notes :)

**Pull request merged**